### PR TITLE
chore: add int VID for NebulaGraph support, schema comment parsing

### PR DESCRIPTION
**Pull request by wey-gu**
# Description

- This small change helps those non-KG graph data to be enabled by GraphRAG and Text2Cypher, where the vid could be numeric and even typed as int64. Previously it assumed we were running a KG that vertex ID was always string.
- Add comment fields in Schema to be parsed, this helped the case when schema field name are short, but comment with rich information in text2cypher

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [x] ~~Added~~ new notebook (that tests end-to-end) [here](https://colab.research.google.com/drive/1Py2JGZHAy70ktreXP4G4mvhVRKQ_nrrf?usp=sharing)
- [ ] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: llama_index/graph_stores/nebulagraph.py [hunk 1]
```diff
@@ -559,17 +608,37 @@ def refresh_schema(self) -> None:
             tag_name = tag.cast()
             tag_schema = {"tag": tag_name, "properties": []}
             r = self.execute(f"DESCRIBE TAG `{tag_name}`")
-            props, types = r.column_values("Field"), r.column_values("Type")
+            props, types, comments = (
+                r.column_values("Field"),
+                r.column_values("Type"),
+                r.column_values("Comment"),
```
Comment (logan-markewich) [inline llama_index/graph_stores/nebulagraph.py:614]: Just curious, is it possible for this line to fail if the graph has no comments?

Comment (wey-gu) [reply llama_index/graph_stores/nebulagraph.py:614]: Great point, but as this query is a DDL read query, the fields are actually fixed, so we could assume the field exists anyway, The only possibilities are value typed in Empty or String :-D

**Comment by wey-gu** [comment]
> Looks good to me! Tbh I think I might need to try and learn cypher soon, most of this goes over my head 😆

Thanks Logan! Looking forward to your getting started to touch it :-D, It's actually of less keyword to SQL! And GPT-4 knows it well actually!

File: llama_index/graph_stores/nebulagraph.py (hunk 1 above)
Comment (logan-markewich) [reply llama_index/graph_stores/nebulagraph.py:614]: Awesome!

**Pull request merged**